.github/workflows/main.yaml

```diff
@@ -31,6 +31,9 @@ jobs:
       - name: CGL
         run: composer run check:php:cs
 
+      - name: PHPSTAN
+        run: composer run check:php:stan
```

Comment:
this seems like a very odd naming to me... is this some typo3 standard?

Reply:
what exactly is odd about this? How would you call that? I have no idea what that name means I only tried to copy from the cgl check

Reply:
I think he means the "check:php:stan" instead of using "check:php:phpstan". The tool is not named "Stan".

Reply:
we do the same for php:cs?

Reply:
Ah I get the confusion now :-P 

`php:cs` => `php:code-style`
'php:stan => `php:static-analysis` 

I thought this was some kind of odd naming convention to convert all tool names to names without `-` (dash)  or camel casing.

Reply:
But "stan" is no package. it is called "phpstan", so "check:php:phpstan" would be better. And I never thought of the name that that would mean "STatic ANalysis". And I doubt we would switch to psalm (or phan) in the next time. But just my 2 cents. I do not block it.

Reply:
we could use 
php:cs => php:code-style
php:stan => php:static-analysis

That would not be bound to one tool and maybe easier to understand for newbies?

Reply:
This isn't blocking for me either.

But it makes sense to keep the naming inline with the commands used to run this locally

Reply:
Ok so I renamed them for the tools names